README.md

```diff
@@ -66,6 +66,24 @@ assert_eq!(taffy.layout(body_node).unwrap().size.height, 500.0); // This value w
 
 ```
 
+## Benchmarks (vs. [Yoga](https://github.com/facebook/yoga))
```

Comment:
I'm still kinda skeptical how meaningful this is, I think it would be better to compare Yoga with the WASM build of taffy called via JS bindings.
Then you have a statement like "you can replace Yoga by taffy for these performance improvements".

Right now it compares two very different languages so a person using JS can't tell if it would benefit them to switch.

Reply:
I think that it shows that our asymptotic efficiency is better for nested trees, which is very compelling.

Reply:
I don't think the language difference is that significant. We're not comparing native to WASM, the node package binds directly to the native C++ module, and so it is only the overhead of the FFI which would make a difference. And while we make lots of function calls over the FFI bridge to setup the tree, we only make a single function call  during the timed part of the benchmark which we would expect to be insignificant. This is backed up by the 10 node result which completes in a few microseconds and gives us an upper bound on the overhead. As the overhead will be constant across all the benchmarks, and results for the largest trees are taking order of magnitude longer than the smallest trees, we can know that the overhead is not significant in these results.

I'd be more worried about a silly mistake in the benchmark code itself. I think part of the reason I want to get the numbers out there is to cajole others into testing them for us.

My intuition is that taffy is just genuinely faster, but that for realistic tree sizes yoga is already fast enough that it's not the bottleneck (which will likely include things like text layout).

Reply:
I would add: we're also not benchmarking tree creation which we might want to at some point.